homework-2/scss/skout-offer.scss

```diff
@@ -0,0 +1,266 @@
+@import "_variables";
+@import "_template";
+@import "_mixins";
+
+.skout-offer {
+	width: $full-width;
+	padding-bottom: 82px;
+
+	background: $bgc-primery;
+
+	.container {
+		padding: 0;
+
+		.skout-offer-question {
+			@include flex-align (flex, center);
+
+			@include mobile {
+				flex-direction: column-reverse;
+			}
+
+			@include desctop {
+				flex-direction: row;
+			}
+
+			img {
+				@include mobile {
+					width: $full-width;
+				}
+			}
+
+			.skout-offer-text {
+				@include mobile {
+					margin-right: 10px;
+					margin-left: 10px;
+				}
+
+				@include widescreen {
+					margin-right: 135px;
+					margin-left: 130px;
+				}
+
+				h5 {
+					color: $color-text-seconrary;
+
+					@include font-box($aclonica-font, 42px, normal, 54px);
+
+					@include mobile {
+						margin-bottom: 20px;
+						padding-right: 20px;
+
+						font-size: $font-size-large;
+						line-height: 36px;
+					}
+
+					@include desctop {
+						margin-bottom: 40px;
+
+						font-size: 42px;
+						line-height: 54px;
+					}
+
+					@include widescreen {
+						padding-right: 0;
+					}
+				}
+
+				p {
+					color: $color-text-seconrary;
+
+					@include font-box($aclonica-font, 16px, normal, 24px);
+
+					@include mobile {
+						font-size: 13px;
+						line-height: $line-size-large;
+					}
+
+					@include desctop {
+						font-size: 16px;
+						line-height: 24px;
+					}
+				}
+			}
+		}
+
+		.skout-offer-accepted {
```

Comment:
links are not in one line 
![image](https://user-images.githubusercontent.com/84564899/122099130-537b6780-ce1a-11eb-8e8e-1d97f639be18.png)